javascript/javascript1/week1/homework of  JSweek1/index.js

```diff
@@ -0,0 +1,49 @@
+//Age-ify ( A future age calculator)
+const yearOfBirth = 1991;
+const yearFuture = 2063;
+const age = yearFuture - yearOfBirth ;
+console.log("You will be " + age + " years old in " + yearFuture)
+
+//Goodboy-Oldboy(a DOG AGE CALCULATOR)
+const dogYearOfBirth = 2017;
+const  dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = dogYearFuture - dogYearOfBirth;
+if(shouldShowResultInDogYears != 70){
+    console.log("Your dog will be ",dogYear," human years old in",dogYearFuture,".")
+} else {
+    console.log("Your dog will be",dogYear,"dog years old in",dogYearFuture )
```

Comment:
what is the difference between human years and dog years?

Reply:
the dog year supposed to be70